.github/workflows/wide_integer.yml

```diff
@@ -7,6 +7,76 @@ on:
   schedule:
     - cron: '0 2 * * *' # run at 2 AM UTC
 jobs:
+  cmake-linux:
+    strategy:
+      fail-fast: false
+      matrix:
+        compiler: [ g++, clang++ ]
```

Comment:
@ckormanyos obviously, to merge this change, you'd need to remove two jobs and add three to the list of required PR jobs.

Reply:
> to merge this change, you'd need to remove two jobs and add three to the list of required PR jobs

Done. Thank you @johnmcfarlane.